src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -685,7 +685,7 @@
                                    (pivot-opts->pivot-spec (merge {:pivot-cols []
                                                                    :pivot-rows []}
                                                                   pivot-export-options) ordered-cols))
-              col-names          (common/column-titles ordered-cols (::mb.viz/column-settings viz-settings) format-rows?)
+              col-names          (streaming.common/column-titles ordered-cols (::mb.viz/column-settings viz-settings) format-rows?)
               pivot-grouping-key (qp.pivot.postprocess/pivot-grouping-key col-names)]
           (when pivot-grouping-key (vreset! pivot-grouping-idx pivot-grouping-key))
           (if opts
```

Comment:
I expected a change to `qp.si/stream-options` in this namespace too, but can't find it..

Reply:
Thanks, fixed -- I think I had changed it but it got lost in some rebase shenanigans